Assets/HoloToolkit-Examples/UX/Readme/README_InteractableObjectExample.md

```diff
@@ -1,13 +1,86 @@
-# Interactable Objects Example
+# Interactable Objects & Receiver
 ![Interactable Objects](/External/ReadMeImages/MRTK_InteractableObject.jpg)
-With these example, you can make any object interactable with differentiated visual state for HoloLens gesture input or immersive headset's motion controller input state. For more information please see ['Interactable Object'](https://developer.microsoft.com/en-us/windows/mixed-reality/interactable_object) on Windows Dev Center.
- 
+This example scene demonstrates how to make objects interactable using [CompoundButton](/Assets/MixedRealityToolkit/UX/Scripts/Buttons/CompoundButton.cs) script scries. With CompoundButton script, you can make any object interactable with differentiated visual state for the HoloLens' gesture input or immersive headset's motion controller input state. For more information please see ['Interactable Object'](https://developer.microsoft.com/en-us/windows/mixed-reality/interactable_object) on Windows Dev Center.
+
+The scene also demonstrates how to use [Receiver](/Assets/MixedRealityToolkit/UX/Scripts/Receivers/InteractionReceiver.cs) to handle events comging from multiple objects, in a single script. This is especially useful when you have multiple buttons or interactable objects in a scene. 
  
+In the scene [InteractableObject_Examples.unity](/Assets/MixedRealityToolkit-Examples/UX/Scenes/InteractableObjectExample.unity), you will be able to find various combinations of 'CompoundButton' scripts. To create your own Interactable Object, you can combine different types of 'CompoundButton' scripts. It is designed to support various types of Interactable Object in flexible way.
+
+### [Compound Button](/Assets/MixedRealityToolkit/UX/Scripts/Buttons/CompoundButton.cs) ###
```

Comment:
All of the links referencing the `MixedRealityToolkit` folder will be broken in the Stabilization branch, since the namespace/folder rename isn't there.